guilded/ext/commands/cog.py

```diff
@@ -197,6 +194,12 @@ def _cog_special_method(func):
 class Cog(metaclass=CogMeta):
     """The base class that all cogs must inherit from.
 
+        finally:
+            try:
+                self.cog_unload()
+            except Exception:
+                pass
```

Comment:
Did you mean to put this in a docstring?

Reply:
Sorry! I am not sure how this gets here, but seems to be a random accidental mid-click buffer pasting result. Removed!